modules/channel-messenger/src/backend/messenger.ts

```diff
@@ -30,11 +29,13 @@ export class MessengerService {
     const config = (await this.bp.config.getModuleConfig('channel-messenger')) as Config
 
     if (!config.verifyToken || config.verifyToken.length < 1) {
-      throw new Error('You need to set a non-empty value for "verifyToken" in the *global* messenger config')
+      throw new Error(
+        'You need to set a non-empty value for "verifyToken" in data/global/config/channel-messenger.json'
+      )
     }
 
     if (!config.appSecret || config.appSecret.length < 1) {
-      throw new Error(`You need to provide your app's App Secret in the *global* messenger config`)
+      throw new Error(`You need to provide your app's App Secret in data/global/config/channel-messenger.json`)
```

Comment:
Would be nice to show the name of the variable also (like verifytoken)! Or/and better, appSecret and verifyToken should be be there by default with an empty string

Also, you could change the length check for  !config.appSecret.length instead of using the <

Reply:
> should be there by default
Not sure what the empty string would change